### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by WeroCode** [itisnajim/SocketIOUnity#14]
- Unity 2022.1.2f1

![image](https://user-images.githubusercontent.com/101685731/170762491-067b602c-be32-4960-b5a0-10ce3920a44b.png)

installed the latest version through Package Manager

**Comment by itisnajim** [comment]
newer versions of unity not supported yet, 
try this url instead to install (with the #v1.0.0 suffix)
https://github.com/itisnajim/SocketIOUnity.git#v1.0.0

if not working, choose a lower version of unity and install via
https://github.com/itisnajim/SocketIOUnity.git

**Comment by jingfu-wei** [comment]
Actually errors shown above do not relate to versions of unity. It was due to the lack of Reactive module of your local .Net environment. Reactive can be found on NuGet. 

I had the same problem with the latest version, and my I am using Unity 2021.3. Eventually, I resolved it by installing Reactive with [NuGetForUnity](https://github.com/GlitchEnzo/NuGetForUnity). Once you install Reactive to your project, errors that shown in the screenshot should be gone after you restart the editor.
